.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+go:
+ - 1.6
```

Comment:
1.6 is just what I am using. We can go with 1.7.4. `release` should cover the latest versions too.